### PR TITLE
tests: invert the conditions for checking modular vs monolithic daemon distro

### PR DESCRIPTION
**Pull request by KKoukiou**
We should have a list that shrinks - not grows

### CODE REVIEW

**Comment by martinpitt** [comment]
Thanks!

> don't have m.image command

`m.image` should totally work, and this PR still relies on it. I suggest to just drop this sentence from the commit message, as it's confusing.

You need to force-push for the pycodestyle error anyway.

File: test/browser/browser.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ systemctl start firewalld
 firewall-cmd --add-service=cockpit --permanent
 firewall-cmd --add-service=cockpit
 
-if grep -Eq 'PLATFORM_ID=.*(f35)' /etc/os-release; then
+if grep -Eq 'PLATFORM_ID=.*(f35|f36|el9)' /etc/os-release; then
```
Comment (martinpitt) [inline test/browser/browser.sh:60]: I think we should invert that too, and *not* do it on f34 and el8. But that can also be a follow-up the next time we have to touch this file.

**Review by martinpitt — DISMISSED** [review]
LGTM, bots have the last word.

**Comment by martinpitt** [comment]
[This flake](https://logs.cockpit-project.org/logs/pull-449-20211111-081011-0b7b03c9-ubuntu-2004/log.html#4) is fixed by #448. Let's rebase this PR after that lands, to not totally ruin our test stats.

**Review by martinpitt — APPROVED** [review]

**Comment by martinpitt** [comment]
Branch was out of date, so packit srpm build failure. Here it actually matters, so I pushed a rebase.

**Review by martinpitt — APPROVED** [review]
Thanks! The packit f35 failure is something entirely different (timeout? VM running out of memory?) but the tests until the hang look good.

**Comment by KKoukiou** [comment]
tests succeeded also downstream.

**Pull request merged**